README.md

```diff
@@ -38,50 +38,55 @@
 
-NewPipe supports multiple services. Our [docs](https://teamnewpipe.github.io/documentation/) provide more info on how a new service can be added to the app and the extractor. Please get in touch with us if you intend to add a new one. Currently supported services are:
+If you intend to add a new service, please get in touch with us first! Our [docs](https://teamnewpipe.github.io/documentation/) provide more information on how a new service can be added to the app and to the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor).
 
-* YouTube
-* SoundCloud \[beta\]
-* media.ccc.de \[beta\]
-* PeerTube instances \[beta\]
-* Bandcamp \[beta\]
+## Description
+
+NewPipe works by fetching the required data from the official API (e.g. PeerTube) of the service you're using. If the official API is restricted (e.g. YouTube) for our purposes, or is proprietary, the app parses the website or uses an internal API instead. This means that you don't need an account on any service to use NewPipe.
+
+Also, since they are free and open source software, neither the app nor the Extractor use any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
+
+### Features
+
+* Watch videos and listen to audios (yes, you can listen to YouTube in the background; it will only load and play the audio stream! Data-saving FTW!) 
+* Popup mode (floating player, aka Picture-in-Picture) (for all you multi-taskers out there)
+* Watch live streams
+* Show/hide subtitles/closed captions
+* Search videos and audios (on YouTube, you can specify the content language as well)
+* Enqueue videos (and optionally save them as local playlists)
+* Show/hide general information about videos (such as description and tags)
+* Show/hide next/related videos
+* Show/hide comments
+* 1080p/2K/4K support
+* Search videos, audios, channels, playlists and albums
+* Browse videos and audios within a channel
+* Subscribe to channels (yes, without logging into any account!)
+* Get notifications about new videos from channels you're subscribed to
+* Create and edit channel groups (for easier browsing and management)
+* Browse video feeds generated from your channel groups
+* View and search your watch history
+* Search and watch playlists (these are remote playlists, which means they're fetched from the service you're browsing)
+* Create and edit local playlists (these are created and saved within the app, and have nothing to do with any service)
+* Download videos
+* Download audios
+* Download subtitles/closed captions
+* Open in Kodi
+* Watch/Block age-restricted material
+* Orbot/Tor support (not within the app yet, but works when used system-wide)
```

Comment:
But I mean, doesn't every app?

Reply:
Some apps leak info despite having Orbot enabled, I think.

Reply:
But it would be good if someone with the knowledge chimes in.

Reply:
I think there is no reason to mention this, as we provide no special integration with Orbot or Tor.

https://www.stoutner.com/webrtc/ may be relevant.

Reply:
Alright. Let's remove it.

Reply:
@opusforlife2
```suggestion
```